client/src/components/v3/DASRecording.module.css

```diff
@@ -0,0 +1,11 @@
+.label {
+    font-weight: bold;
+}
+
+.device {
+    margin-left: 8px;
+}
+
+.pill {
+    margin-left: 5px;
+}
```

Comment:
Because this is so little CSS is it worth making it inline? Like `style={{marginLeft: 8}}`

Reply:
I could see that being more concise. Though there is also an argument to be made on consistency as we use css modules across the codebase. Happy to discuss though, thoughts @hallgchris ?